genai-perf/genai_perf/metrics/statistics.py

```diff
@@ -131,6 +131,8 @@ def _add_units(self, key) -> None:
             self._stats_dict[key]["unit"] = "ms"
         elif key == "request_throughput":
             self._stats_dict[key]["unit"] = "requests/sec"
+        elif key == "request_goodput":
```

Comment:
You can combine this with the previous if condition for conciseness.

Reply:
Updated